utils/utils.py

```diff
@@ -124,15 +126,40 @@ def render_dialog(text: str, character: str, background: str = 'camp') -> BytesI
 
     return result
 
-def status_embed(text : str, status = True) -> discord.Embed:
+
+def status_embed(text: str, status=True) -> discord.Embed:
     embed = discord.Embed(description=text)
 
-    if status :
+    if status:
         embed.title = "✅"
-        embed.color = discord.Color.green()
-    else :
+        # color is not modifiable
+        embed.colour = discord.Color.green()
+    else:
         embed.title = "Oops"
-        embed.color = discord.Color.red()
+        embed.colour = discord.Color.red()
 
     return embed
-    
\ No newline at end of file
+
+
+def buddy_name_to_color(name: str) -> int:
+    items = {
+        'keitaro': 0x39fe19,
+        'hiro': 0xff6600,
+        'yoichi': 0xb11aff,
+        'natsumi': 0x1ab1ff,
+        'hunter': 0xf7f63c,
+        'taiga': 0xe81615,
+        'aiden': 0x5d8c6b,
+        'goro': 0x39fe19,
+        'yoshinori': 0xc07c35,
+        'yuri': 0xff73bb,
+        'seto': 0x9dd5e6,
+        'felix': 0xbdd7e6,
+        'eduard': 0xd37fb7,
+        'lee': 0x00aa6c,
+        'kieran': 0xe7ab0d,
+        'naoto': 0x227abe,
+    }
```

Comment:
`assets/chars.json` has every color in hex, no need to store it multiple times. Use the JSON, or port the JSON to a Python-compatible format as we did with #14 Fix/Convert's dictionary.

Reply:
Thanks for the tip